### PR TITLE
Removed TypeMsg constants

### PR DESCRIPTION
**Pull request by alizahidraja**
## Description

closes: #2434

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/core/04-channel/keeper/keeper_test.go [hunk 1]
```diff
@@ -105,13 +106,13 @@ func (suite *KeeperTestSuite) TestGetAllChannelsWithPortPrefix() {
 			name:             "transfer channel is retrieved with prefix",
 			prefix:           "tra",
 			allChannels:      allChannels,
-			expectedChannels: []types.IdentifiedChannel{types.NewIdentifiedChannel(transfertypes.TypeMsgTransfer, ibctesting.FirstChannelID, types.Channel{})},
+			expectedChannels: []types.IdentifiedChannel{types.NewIdentifiedChannel(MsgTransfer, ibctesting.FirstChannelID, types.Channel{})},
```
Comment (damiannolan) [inline modules/core/04-channel/keeper/keeper_test.go:109]: I think you can use either `transfertypes.PortID` or even [`ibcmock.PortID`](https://github.com/cosmos/ibc-go/blob/main/testing/mock/mock.go#L25) for the first argument (portID) here instead, and remove the `MsgTransfer` const above.

**Review by damiannolan — COMMENTED** [review]
Thanks for this @alizahidraja! 

Left one comment regarding `portID` arg in 04-channel tests.

File: modules/core/04-channel/keeper/keeper_test.go (hunk 1 above)
Comment (alizahidraja) [reply modules/core/04-channel/keeper/keeper_test.go:109]: Thanks for the review! Will update this

**Comment by alizahidraja** [comment]
Need help, can't figure out why that one test is failing 

`Check Markdown links / markdown-link-check (pull_request) Failing`

**Comment by damiannolan** [comment]
> Need help, can't figure out why that one test is failing
> 
> `Check Markdown links / markdown-link-check (pull_request) Failing`

All good. Looks like a dead link in some older migration docs: https://github.com/cosmos/ibc-go/pull/2488

**Pull request merged**